news/1983.doc

```diff
@@ -0,0 +1 @@
+Remove VCS URLs pertaining to the Git protocol from docs
```

Comment:
I'd rephrase this to clarify what has been changed.

```suggestion
Emphasize that VCS URLs using ``git``, ``git+git`` and ``git+http`` are insecure due to {???}
```

Reply:
Changed to `Emphasize that VCS URLs using git, git+git and git+http are insecure due to
lack of authentication and encryption`